### DISCUSSION TITLE
Renovate bot removes labels

### DISCUSSION THREAD

**Discussion by Neenu1995** [renovatebot/renovate#11802] [Request Help]
Hi folks,

All java libraries in the googleapis have Dependency Dashboard enabled. There is tooling in place to add the product labels in issues and PRs in the respective repos. But renovate bot keeps removing the label, our tooling puts it back and this keeps happening in an endless loop.
This could be related to [this previous issue](https://github.com/renovatebot/renovate/discussions/10798), which was resolved for the java libraries with [this](https://github.com/googleapis/java-dns/pull/453) change to the renovate configuration. Is there a possible work around or fix for the current issue?

Sample Issues:
https://github.com/googleapis/java-dns/issues/193
https://github.com/googleapis/java-eventarc/issues/15
https://github.com/googleapis/java-dialogflow/issues/297

@danoscarmike @lesv

**Comment by rarkins** [comment]
If you want to control your own labels, you need to remove the `dependencyDashboardLabels` setting in your repos. Per the docs here: https://docs.renovatebot.com/configuration-options/#dependencydashboardlabels

![image](https://user-images.githubusercontent.com/6311784/133929323-79655e2e-e63c-43d3-bc81-53932a535cf1.png)

**Reply by Neenu1995** [reply]
Thank you for the quick response :)
